src/responder/ifp/ifp_iface.xml

```diff
@@ -5,8 +5,8 @@
         <annotation value="iface_ifp" name="org.freedesktop.DBus.GLib.CSymbol"/>
 
         <method name="Ping">
-            <!-- manual argument parsing, raw handler -->
-            <annotation name="org.freedesktop.sssd.RawHandler" value="true"/>
+            <arg name="ping" type="s" direction="in" />
+            <arg name="pong" type="s" direction="out" />
```

Comment:
I think it was enough to add arguments to make  "introspect"  happy
```
diff --git a/src/responder/ifp/ifp_iface.xml b/src/responder/ifp/ifp_iface.xml
index ce071bb99..d04ddf035 100644
--- a/src/responder/ifp/ifp_iface.xml
+++ b/src/responder/ifp/ifp_iface.xml
@@ -7,6 +7,8 @@
         <method name="Ping">
             <!-- manual argument parsing, raw handler -->
             <annotation name="org.freedesktop.sssd.RawHandler" value="true"/>
+            <arg name="ping" type="s" direction="in" />
+            <arg name="pong" type="s" direction="out"/>
         </method>
 
         <!-- SSSD components -->
```
It generated simpler diff but that works for me as well.

Reply:
It is just in case we want to have an example how to use `sssd.Rawhander`
@pbrezina it's up to you :-)